api/proto/hapi/generate.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+SRC_DIR=.
+GO_DST_DIR=go/
+JS_DST_DIR=js/
+TS_DST_DIR=ts/
+
+rm -rf $GO_DST_DIR $JS_DST_DIR $TS_DST_DIR
+mkdir  $GO_DST_DIR $JS_DST_DIR $TS_DST_DIR
+
+for proto in *.proto
+do
+    set -x
+    protoc -I=$SRC_DIR --go_out=$GO_DST_DIR $SRC_DIR/$proto
+    protoc -I=$SRC_DIR --grpc-web_out=import_style=typescript,mode=grpcwebtext:$TS_DST_DIR $proto
+    protoc -I=$SRC_DIR --js_out=import_style=commonjs:$JS_DST_DIR  --grpc-web_out=import_style=commonjs,mode=grpcwebtext:$JS_DST_DIR $proto
```

Comment:
I'm not sure we need JS in addition to TS?

Reply:
I wasn't sure which you needed. I can easily remove it if you don't want it. If we are going to publish it on npm's main repo, does it make sense to generate it? I'm not really a Node guy, so I'm not super familiar with how these things interoperate.

Reply:
Per discussion, not sure what to do here. Going to kick this down the road to the packaging ticket. https://czi-tech.atlassian.net/browse/CCIE-1649